Hardware/Hardware/CardReader/CardReaderService.swift

```diff
@@ -49,10 +49,6 @@ public protocol CardReaderService {
     /// Captures a payment after collecting a payment method succeeds.
     func capturePayment(_ parameters: PaymentIntentParameters) -> AnyPublisher<PaymentIntent, Error>
 
-    /// Cancels a a PaymentIntent
```

Comment:
Why this change?

Reply:
We don't really need it at this point, it looks like I was a bit trigger happy with the delete button. It shouldn't have made it to this PR, I'll undo and log a separate ticket for cleaning up the service

Reply:
#4068